pkgs/development/python-modules/bidict/default.nix

```
@@ -0,0 +1,42 @@
+{ lib, buildPythonPackage, fetchPypi
+, setuptools_scm
+, sphinx
+, hypothesis
+, py
+, pytest
+, pytest-benchmark
+, sortedcollections
+, sortedcontainers
+}:
+
+buildPythonPackage rec {
+  pname = "bidict";
+  version = "0.17.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1icj0fnfx47n6i33pj5gfrmd1rzpvah1jihhdhqiqx2cy9rs6x4c";
+  };
+
+  # hypothesis-pytest is not needed
+  patches = [ ./hypothesis_pytest.patch ];
+
+  nativeBuildInputs = [ setuptools_scm ];
+  propagatedBuildInputs = [ sphinx ];
+
+  checkInputs = [
+    hypothesis
+    py
+    pytest
+    pytest-benchmark
+    sortedcollections
+    sortedcontainers
+  ];
+
+  meta = with lib; {
+    homepage = https://github.com/jab/bidict;
+    description = "Efficient, Pythonic bidirectional map data structures and related functionality";
+    license = licenses.mpl2;
```

```suggestion
    license = licenses.mpl20;
```

This will fix eval. Licenses attrs can be seen https://github.com/NixOS/nixpkgs/blob/master/lib/licenses.nix

---

Weird, it's been building fine for me locally.

---

This is actually a special check that ofborg does. The results of the checks can be seen at the bottom of any pr.